pkgs/applications/backup/vdmfec/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "vdmfec_${version}";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "http://deb.debian.org/debian/pool/main/v/vdmfec/${name}.orig.tar.gz";
```

Comment:
Why not `http://members.tripod.com/professor_tom/archives/${name}.tgz`?
You can have both, if you want:
```
fetchurl {
  urls = [
    "http://members.tripod.com/professor_tom/archives/${name}.tgz"
    "http://deb.debian.org/debian/pool/main/v/vdmfec/${name}.orig.tar.gz"
  ];
  # ...
}

Reply:
Actually, this won't work, tarballs are different:
```
# nix-prefetch-url "http://cdn-fastly.deb.debian.org/debian/pool/main/v/vdmfec/vdmfec_1.0.orig.tar.gz"
path is '/nix/store/970nhjvk0hzsahy5avgc1l48bqslq32p-vdmfec_1.0.orig.tar.gz'
1phb1qax5adkcdiy4q18814yc86ayfsm6318arlcxdd2g2rnsjz4
# nix-prefetch-url "http://members.tripod.com/professor_tom/archives/vdmfec-1.0.tgz"
path is '/nix/store/miqr6a3zwv477h25d945xfy107qn336a-vdmfec-1.0.tgz'
0i7q4ylx2xmzzq778anpkj4nqir5gf573n1lbpxnbc10ymsjq2rm
```
We need to pick one.